spk/python3/Makefile

```diff
@@ -10,7 +10,7 @@ DEPENDS += cross/setuptools cross/pip cross/wheel
 DEPENDS += cross/cffi cross/bcrypt cross/sqlite
 # Cross-compiled wheels
 DEPENDS += cross/lxml cross/pycrypto cross/pycurl cross/pyyaml
-DEPENDS += cross/msgpack-python cross/attr cross/acl cross/lz4
+DEPENDS += cross/msgpack-python cross/ruamel.yaml
```

Comment:
Why removing attr, acl and lz4 from Python 3?
Isn't there any way to load ruamel.yaml (typically as borgbackup package specific wheel) instead of enforcing Python 3 publish (again) ?

Reply:
As far as I know, attr, acl and lz4 are used by no other python3 dependent packages except for borgbackup.
I managed to put those dependencies inside borkgbackup package itself.
Better check with HomeAssistant spk maintainer, because in my eyes it looks those dependencies are not used.
About ruamel.yaml, I was not able to put it inside borgbackup package (any idea?).